src/applications/vaos/components/calendar/CalendarWidget.unit.spec.jsx

```diff
@@ -567,4 +568,51 @@ describe('VAOS Component: CalendarWidget', () => {
       'vaos-calendar__disabled',
     );
   });
+
+  it('should show validation alert when max selections+1 dates selected for requests', async () => {
+    // Arrange
+    const now = new Date();
+    const selectedDates = [
+      format(addDays(now, 1), DATE_FORMATS.ISODateTimeLocal),
+      format(addDays(now, 2), DATE_FORMATS.ISODateTimeLocal),
+      format(addDays(now, 3), DATE_FORMATS.ISODateTimeLocal),
+      format(addDays(now, 4), DATE_FORMATS.ISODateTimeLocal),
+    ];
+
+    // Act
+    const screen = renderWithStoreAndRouter(
+      <CalendarWidget maxSelections={3} value={selectedDates} />,
+      {},
+    );
+
+    expect(
+      screen.queryByText(
+        'You can only select up 3 times for your appointment. Deselect the 4th time to continue.',
```

Comment:
Should the text be: 'You can only select 3 times for your appointment. Deselect [#] times to continue.' or 'You can only select up to 3 times for your appointment. Deselect the 4th time to continue.'

Reply:
Good catch!